journal-of-paleontology.csl

```diff
@@ -179,7 +179,7 @@
                 <choose>
                   <if is-numeric="volume">
                     <group delimiter=" ">
-                      <text value="Vol. "/>
```

Comment:
I'd much rather do this properly with label so here `<label variable="volume" form="short"/>` and then define
```
<locale>
  <terms>
    <term name="volume" form="short">v.</term>
  </terms>
</locale>
```

after line 20. above

Reply:
It's `form="short"`; you have `format="short"` which doesn't exist, so CSL reverts to the default, which is long.

Reply:
Whoops! Thanks.  Looking good now.